deploy/helm-chart/README.md

```diff
@@ -0,0 +1,135 @@
+# NGINX Kubernetes Gateway Helm Chart
+
+## Introduction
+
+This chart deploys the NGINX Kubernetes Gateway in your Kubernetes cluster.
+
+## Prerequisites
+
+- [Helm 3.0+](https://helm.sh/docs/intro/install/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+> Note: NGINX Kubernetes Gateway can only run in the `nginx-gateway` namespace. This limitation will be addressed in
+the future releases.
+
+## Installing the Chart
+
+> Note: The Gateway API resources from the standard channel (the CRDs and the validating webhook) must be installed
+before deploying NGINX Kubernetes Gateway. By default, they will be installed by the Chart if not already
+present in the cluster. If they are already installed in your cluster, please ensure they are the correct version as
+supported by the NGINX Kubernetes Gateway - [see the Technical Specifications](../../README.md#technical-specifications).
+Helm will not upgrade CRDs - to do so manually, see
+[Upgrading the Gateway API resources](#upgrading-the-gateway-resources).
+
+### Installing the Chart from the OCI registry
```

Comment:
nit: inconsistent section titles capitalization.
similar issue in a few other titles too
```suggestion
### Installing the Chart from the OCI Registry
```